cluster/ubuntu/util.sh

```diff
@@ -231,6 +231,11 @@ EOF
 }
 
 function create-kubelet-opts() {
+  if [ -n "$6" ] ; then
+      cni_opts=" --network-plugin=cni --network-plugin-dir=/etc/cni/net.d"
```

Comment:
I know you didn't add these args, but can you please describe what they are above the function? eg: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L317